app/views/layouts/application.html.erb

```diff
@@ -7,8 +7,9 @@
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
+  <%= render '/layouts/navbar' %>
+  <%= yield %>
+  <%= render '/layouts/footer' %>
 
```

Comment:
but i don't know that we want it in the full application. on the landing page we will not want the nav bar to show.